settings.gradle.kts

```diff
@@ -29,3 +29,4 @@ include(":integration-tests:ksp")
 include(":integration-tests:module")
 include(":integration-tests:ksp-companion")
 include(":integration-tests:jmh")
+include(":target-component-accessor-compiler:ksp")
```

Comment:
seems like it would be easy to add the annotation but forget to include the extra compiler dependency. It's simple enough that maybe we don't need to split it out?

Reply:
I didn't do that because there could be problems if the processor is added to the common configuration, but maybe that's fine because using this annotation when the processor is in the common configuration is already a mistake. Hopefully the error (`actual` functions generated in the common source set) will be clear enough that the user realizes what mistake they made.

Reply:
gotcha, in that case failing loudly may actually be better than doing nothing assuming the error makes it obvious enough which sounds like it might